myshop/cart/views.py

```diff
@@ -25,3 +25,11 @@ def cart_add(request, product_id):
 def cart_detail(request):
     cart = Cart(request)
     return render(request, "cart/detail.html", {"cart": cart})
+
+
+@require_POST
+def cart_remove(request, product_id):
+    cart = Cart(request)
+    product = get_object_or_404(Product, id=product_id)
+    cart.remove_product(product)
```

Comment:
Test case for remove_product() alone

Reply:
Test case for `remove_product` method has already been written in [test_cart.py](https://github.com/Deepak-Testpress/shopping_app/blob/98be6ff82b792c44531a2231e7acb4d5f5532b1e/myshop/tests/cart/test_cart.py#L104)